zipkin-server/README.md

```diff
@@ -154,7 +154,7 @@ Defaults to true
 * `QUERY_LOOKBACK`: How many milliseconds queries can look back from endTs; Defaults to 24 hours (two daily buckets: one for today and one for yesterday)
 * `STORAGE_TYPE`: SpanStore implementation: one of `mem`, `mysql`, `cassandra`, `elasticsearch`
 * `COLLECTOR_SAMPLE_RATE`: Percentage of traces to retain, defaults to always sample (1.0).
-* `AUTOCOMPLETE_KEYS`: list of span tag keys which will be returned by the `/api/v2/autocompleteTags` endpoint
+* `AUTOCOMPLETE_KEYS`: list of span tag keys which will be returned by the `/api/v2/autocompleteTags` endpoint; Tag keys should be comma separated, e.g. "request_id,instance_id,env"
```

Comment:
request_id is a bad example because it is high cardinality. otherwise looks good

Reply:
Just changed it: https://github.com/apache/incubator-zipkin/pull/2594/commits/9cb9b856b79d777f74ad4afd659612d1df55b460